client/gutenberg/extensions/presets/jetpack/editor.js

```diff
@@ -18,6 +19,14 @@ import * as TiledGalleryBlock from 'gutenberg/extensions/tiled-gallery';
 import * as VRBlock from 'gutenberg/extensions/vr';
 import { isEnabled } from 'config';
 
+export async function getExtensions() {
+	const promises = extensionSlugs.production.map( slug =>
+		import( '../../' + slug ).then( ( { name, settings } ) => ( { name, settings } ) )
+	);
+
+	return await Promise.all( promises ); // Y U NO RETURN ARRAY BUT PROMISE?
```

Comment:
Try adding `console.log( getExtensions() )` below this function definition -- it'll spit out a promise instead of an array of block definitions (i.e. `{ name, settings }` pairs). This means we can't simply use `getExtensions` inside of `refreshRegistrations()` to iterate over. (I've also tried making that function `async` and to `await` `getExtensions()`, but no dice).

Does anyone have a better clue about promises/async functions?

Reply:
> I've also tried making that function `async` and to `await` `getExtensions()`, but no dice

Hmm, on second try, this seems to do the trick. I'll push a commit.